### PR TITLE
Add travis

### PR DESCRIPTION
**Pull request by nickw444**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -16,19 +16,12 @@
     "quotes": [2, "single", "avoid-escape"],
     "strict": [2, "never"],
     "semi-spacing": [2, {"before": false, "after": true}],
-    "react/jsx-uses-react": 2,
```
Comment (johnwiseheart) [inline .eslintrc]: Do the tests work if you remove this? I tried to remove it but all the tests failed for me.

Comment (nickw444) [inline .eslintrc]: Worked for me :)

Comment (nickw444) [inline .eslintrc]: Linting failed with this because it couldn't find the plugin

Comment (johnwiseheart) [inline .eslintrc]: Sweet I must have screwed up somehow

Comment (nickw444) [inline .eslintrc]: Plus like, we're not using react/JSX in here... so 👍

Comment (nickw444) [inline .eslintrc]: Something to do with global eslint probable?

**Pull request merged**